lib/api/2.0/notification.js

```diff
@@ -30,7 +36,124 @@ var notificationProgressPost = controller(function(req, res){
     });
 });
 
+/**
+ * @api {post} /api/2.0/notification/alerts
+ * @apiDescription Receives Redfish alerts, refactors the message and publishs it into amqp
+ * @apiName notification post
+ * @apiGroup notification
+ */
+
+var notificationAlertsPost = controller({success: 201}, function(req,res) {
+    /* The alert would look like the following:
+     {
+     "type": "node",
+     "action": "alerts",
+     "typeId": "58d94cec316779d4126be134",
+     "data": {
+         "Context": "context string",
+         "EventId": "8689",
+         "EventTimestamp": "2017-03-29T15:39:34-0500",
+         "EventType": "Alert",
+         "MemberId": "7e675c8e-127a-11e7-9fc8-64006ac35232",
+         "Message": "The coin cell battery in CMC 1 is not working.",
+         "MessageArgs": ["1"],
+         "MessageArgs@odata.count": 1,
+         "MessageId": "CMC8572",
+         "Severity": "Critical",
+         "sourceIpAddress": "10.240.19.130",
+         "rackhdId": "58d94cec316779d4126be134",
+         "ChassisName": "PowerEdge R630",
+         "ServiceTag": "4666482",
+         "SN": "CN747515A80855"
+         },
+     "severity": "critical",
+     "version": "1.0",
+     "createdAt": "2017-03-29T19:44:12.972Z"
+     }
+     */
+
+    return Promise.try(function(){
+        var ip = req.headers['x-forwarded-for'] ||
+            req.connection.remoteAddress ||
+            req.socket.remoteAddress ||
+            req.connection.socket.remoteAddress;
+        var amqpMessage = {};
+        var deviceInfoCatalog ={};
+        var bmcCatalog = {};
+        var alert = {};
+
+        return new Promise(function(resolve, reject) {
+            var parse = parser.urlencoded({ extended: false });
+            parse(req, res, function (err) {
+                if (err) {
+                    return reject(err);
+                }
+                    resolve();
+            });
+        })
+            .then(function() {
+            return waterline.catalogs.findOne({source: "bmc", "data.IP Address": ip})
+                .then(function(data) {
+                    if (data !== undefined && data.hasOwnProperty("node") && data.node !== ""){
+                         bmcCatalog = data;
+                        //Format the amqp message by adopting the rackHD notification standard
+                        amqpMessage.type = "node";
+                        amqpMessage.action = "alerts";
```

Comment:
Because we use the `action` property in the routing key, I avoiding the use of dot notation because it will add confusion.
http://rackhd.readthedocs.io/en/latest/rackhd/event_notification.html?highlight=event#events-payloads .
`Routing Key <type>.<action>.<severity>.<typeId>.<nodeId>`
With your suggestion, a person might confuse  `alert` as a `severity`.
Do you have any one word suggestion ?